bedrock/settings/base.py

```diff
@@ -1407,10 +1411,15 @@ def get_app_name(hostname):
     'tagmanager.google.com',
     'www.youtube.com',
     's.ytimg.com',
+    'cdn-3.convertexperiments.com',
+    'app.convert.com',
+    '1003350.track.convertexperiments.com',
+    '1003343.track.convertexperiments.com',
```

Comment:
```
    '1003350.track.convertexperiments.com',
    '1003343.track.convertexperiments.com',
```
I saw that these numbers correlate to our project directory IDs in convert. Is this secure? Do those ID numbers need to be secret?

Reply:
There's nothing in this PR that needs to remain secret. This is all visible information that's available to anyone who opens dev tools, much like our Google Analytics ID.